src/rules/selector-descendant-combinator-no-non-space/README.md

```diff
@@ -0,0 +1,30 @@
+# selector-descendant-combinator-no-non-space
+
+Disable non-spaces characters of the descendant combinator in descendant selectors.
+
+```css
+   .selector1 .selector2 .selector3 { color: pink; }
+/**          ↑          ↑
+ * These are descendant combinators */
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+.selector1 
+.selector2 {}
+```
```

Comment:
A little more explanation is needed because "non-space" is not a self-evident term.